packages/space-opera/src/components/model_viewer_snippet/model_viewer_snippet.ts

```diff
@@ -45,7 +45,7 @@ import {applyRelativeFilePaths, getExtraAttributes} from './reducer.js';
  */
 @customElement('me-export-panel')
 export class ExportPanel extends ConnectedLitElement {
-  @property({type: String}) header = '';
+  @property({type: Boolean}) header = false;
```

Comment:
Dumb question, what is header used for in ExportPanel?

Reply:
It's the snippet-only section that's shown in the edit and camera tabs. This whole chunk of code is  quite  confusing to me; I tried to refactor it, but it was becoming too large a change to be worthwhile just now.